plugins/module_utils/network/vyos/vyos.py

```
@@ -27,12 +27,14 @@
 #
 from __future__ import absolute_import, division, print_function
 
+
 __metaclass__ = type
 import json
 
 from ansible.module_utils._text import to_text
 from ansible.module_utils.connection import Connection, ConnectionError
```

Ruff (0.16.3)

[error] Import `ConnectionError` is shadowing a Python builtin

(A004)